src/Components/custom/Custom.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { useQueryParam, NumberParam, StringParam, ArrayParam, withDefault } from 'use-query-params';
+import View from './View';
+import Edit from './Edit';
+import { useSelector, useDispatch } from 'react-redux';
+import { isDataBeingEditedUpdated, titleUpdated, subtitleUpdated, descriptionUpdated, mainImageKeywordsChanged, bgImageKeywordsChanged } from '../../app/customWebsiteDataSlice';
+
+const Custom = () => {
+  const edit = useSelector(state => state.customWebsiteData.isDataBeingEdited)
+  const bgImageUrl = useSelector(state => state.customWebsiteData.bgImageUrl)
+
+  const [editParam, setEditParam] = useQueryParam('edit', NumberParam)
+  const [titleParam, setTitle] = useQueryParam('title', StringParam)
+  const [subtitleParam, setSubtitle] = useQueryParam('subtitle', StringParam)
+  const [descriptionParam, setDescription] = useQueryParam('description', StringParam)
+  const [mainImageParam, setMainImage] = useQueryParam('mainImage', withDefault(ArrayParam, []))
+  const [bgImageParam, setBgImage] = useQueryParam('bgImage', withDefault(ArrayParam, []))
+
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    if(typeof editParam === 'number') dispatch(isDataBeingEditedUpdated(editParam == 1 ? true : false))    
```

Comment:
I am not a fan of one line if statements. Can these split out as two lines.
ideally managed with eslint config if that supports it

Reply:
Done